<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>180</y>
      <width>561</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Style for QListWidget */
#music_list {
    background-color: #f0f0f0;
    padding: 5px;
}

/* Style for QListWidget items */
#music_list::item {
    background-color: window;  /* Match window background color */
    padding: 5px;
    color: #000000;  /* Black text color */
}

/* Style for QListWidget items when hovered */
#music_list::item:hover {
    background-color: #c0c0c0;  /* Custom hover background color */
}
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>517</width>
       <height>136</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="current_music">
              <property name="styleSheet">
               <string notr="true">font-size : 30px;</string>
              </property>
              <property name="text">
               <string>Music name</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="prev_btn">
                <property name="styleSheet">
                 <string notr="true">#prev_btn {
    background-color: #3498db; /* Button background color */
    
    border: 1px solid #3498db; /* Button border color */
    padding: 8px 16px; /* Padding around the text */
    font-size: 20px; /* Font size of the text */
    border-radius: 5px; /* Rounded corners */
 	color: black; 	
	font-weight: bold;
}


#prev_btn:hover {
    background-color: #2980b9; /* Button background color on hover */
    border: 1px solid #2980b9; /* Button border color on hover */
}

</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="play_btn">
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">#play_btn {
    background-color: #3498db; /* Button background color */
    border: 1px solid #3498db; /* Button border color */
    padding: 8px 16px; /* Padding around the text */
    font-size: 20px; /* Font size of the text */
    border-radius: 5px; /* Rounded corners */
 	color: black; 	
	font-weight: bold;
}

#play_btn:hover {
    background-color: #2980b9; /* Button background color on hover */
    border: 1px solid #2980b9; /* Button border color on hover */
}</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="next_btn">
                <property name="styleSheet">
                 <string notr="true">#next_btn {
    background-color: #3498db; /* Button background color */   border: 1px solid #3498db; /* Button border color */
    padding: 8px 16px; /* Padding around the text */
    font-size: 20px; /* Font size of the text */
    border-radius: 5px; /* Rounded corners */
 	color: black; 	
	font-weight: bold;
}

#next_btn:hover {
    background-color: #2980b9; /* Button background color on hover */
    border: 1px solid #2980b9; /* Button border color on hover */
}</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QSlider" name="vertical_slider">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="curr_id">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="txt">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>OF</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="length">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QListWidget" name="music_list">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>341</width>
      <height>481</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Style for QListWidget */
#music_list {
   background-color: #222222; /* Dark background color */
    border: 1px solid #333333; /* Dark border */
}

/* Style for QListWidget items */
#music_list::item {
    background-color: #444444; /* Darker item background color */
    padding: 10px;
    color: #ffffff; /* White text color */
    border: 1px solid #555555; /* Slightly lighter border */
   /* Change font style */
    font-family: 'Arial', sans-serif;
    font-size: 14px;
    font-weight: bold;
}

/* Style for QListWidget items when hovered */
#music_list::item:hover {
    background-color: #666666; /* Lighter hover background color */
}

/* Style for scroll bar */
#music_list::vertical {
    border: 1px solid #333333; /* Dark border */
    background: #222222; /* Dark background color */
}

#music_list::handle:vertical {
    background: #555555; /* Lighter handle color */
}

#music_list::handle:vertical:hover {
    background: #666666; /* Lighter hover handle color */
}
</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
